tests/measureSources.py

```diff
@@ -294,6 +302,31 @@ def testPixelFlags(self):
             source,
             exp,
         )
+        # Test that if there is no center and centroider that the object should look at the footprint
+        plugin, cat = makePluginAndCat(measBase.PixelFlagsAlgorithm, "test", control)
+        # The first test should raise exception because there is no footprint
+        source = cat.makeRecord()
+        self.assertRaises(
+                lsst.pex.exceptions.RuntimeError,
+                plugin.measure,
+                source,
+                exp,
+        )
+        # The second test will raise an error because no peaks are present
+        source.setFootprint(afwDetection.Footprint(afwGeom.Point2I(afwGeom.Point2D(x+x0, y+y0)),5))
```

Comment:
Spacing around operators.

Also, it seems like a bad idea to rely on `x` being what you left it at the end of the loop above. Next time somebody extends that loop, these tests will spontaneously break.